src/core/src/sys/interactions.rs

```diff
@@ -61,22 +58,28 @@ pub struct Interactions {
     angles: BTreeMap<AngleKind, Vec<Box<AnglePotential>>>,
     /// Dihedral angles potentials
     dihedrals: BTreeMap<DihedralKind, Vec<Box<DihedralPotential>>>,
+}
+
+/// The `Interaction` type hold all data about the potentials in the system.
+///
+/// Its main role is to store and provide access
+#[derive(Clone)]
+pub struct Interactions {
+    /// Local intreactions
+    pub local: LocalInteractions,
```

Comment:
Instead of having a pub field, how about implementing `Deref<Target=LocalInteractions>`?